script/publish-release

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const path = require('path')
+const glob = require('glob')
+const publishRelease = require('publish-release')
+const CONFIG = require('./config')
+
+console.log(`Publishing GitHub release ${CONFIG.computedAppVersion}`)
+
+const yargs = require('yargs')
+const argv = yargs
+  .usage('Usage: $0 [options]')
+  .help('help')
+  .describe('assets-path', 'Path to the folder where all release assets are stored')
+  .wrap(yargs.terminalWidth())
+  .argv
+
+let assetsPath = argv.assetsPath || path.join(CONFIG.repositoryRootPath, 'out')
+let assets = glob.sync(path.join(assetsPath, '*(*.exe|*.zip|*.nupkg|*.tar.gz|*.rpm|*.deb|RELEASES*)'))
+
+publishRelease({
+  token: process.env.GITHUB_TOKEN,
+  owner: 'atom',
+  repo: CONFIG.channel !== 'nightly' ? 'atom' : 'atom-nightly-releases',
```

Comment:
Can you remind me why it makes sense to have a separate repository for the nightly releases?

Reply:
Two reasons:

1. atom/atom's Releases feed will get swamped with nightly releases and it'll be harder to scan through the list to find older stable/beta releases
2. My goal was to use Electron's new update service for this and they use the Releases feed as the source of truth for updates.  Seemed to make more sense to have it be a separate repo for now.  Can always change it later!